src/DriverBase.cpp

```diff
@@ -58,10 +65,23 @@ void DriverBase::setAnalogInputEnableInTPDO(int index, bool flag) {
     }
 }
 
+void DriverBase::setEncoderCounterEnableInTPDO(int index, bool flag) {
```

Comment:
the `flag` is the toggles for enable/disable?
```suggestion
void DriverBase::setEncoderCounterEnableInTPDO(int index, bool enabled) {
```